### PR TITLE
Don't make users worry about logs

### PR DESCRIPTION
**Pull request by breeswish**
Signed-off-by: Breezewish <breezewish@pingcap.com>

## What have you changed? (mandatory)

Logs when batch execution cannot be utilized is not a big deal. This PR adjusts the level and the wording of batch execution log outputs, so that user will won't be worried when seeing these logs.

## What are the type of the changes? (mandatory)

- Improvement (change which is an improvement to an existing feature)

### CODE REVIEW

File: src/coprocessor/dag/builder.rs [hunk 1]
```diff
@@ -81,18 +70,14 @@ impl DAGBuilder {
...
+                        Error::Other(box_err!("BatchStreamAggregationExecutor: {}", e))
                     })?;
                 }
                 ExecType::TypeLimit => {}
                 ExecType::TypeTopN => {
                     let descriptor = ed.get_topN();
-                    BatchTopNExecutor::check_supported(&descriptor).map_err(|e| {
-                        Error::Other(box_err!("Unable to use BatchTopNExecutor: {}", e))
-                    })?;
+                    BatchTopNExecutor::check_supported(&descriptor)
```
Comment (BusyJay) [inline src/coprocessor/dag/builder.rs]: Is it necessary to allocate for a common error?

Comment (breeswish) [reply src/coprocessor/dag/builder.rs]: It is not a common error

Comment (BusyJay) [reply src/coprocessor/dag/builder.rs]: Then why people complains about the frequent logs?

Comment (breeswish) [reply src/coprocessor/dag/builder.rs]: I doubt these logs (Enum / Set not supported) comes from previous versions. After #4788 very few cases would report this kind of error.

Comment (breeswish) [reply src/coprocessor/dag/builder.rs]: Additionally one or few allocation is not worth mentioning for a Coprocessor request, which is very heavy already. For example, we have at least two allocations when scanning each row now. Thinking about how many allocations there will be for a table scan request.

**Pull request merged**